apps/glusterfs/device_entry.go

```diff
@@ -581,7 +581,7 @@ func (d *DeviceEntry) DeleteBricksWithEmptyPath(tx *bolt.Tx) error {
 		brick, err := NewBrickEntryFromId(tx, id)
 		if err == ErrNotFound {
 			logger.Warning("Ignoring nonexistent brick [%v] on "+
-				"disk [%v].", id, d.Info.Id)
```

Comment:
the reason I chose to use %v instead of %d is that other IDs are type of string.  And brick id in other messages is presented as %v as well.